RatTracker-WPF/RatTracker-WPF/MainWindow.xaml.cs

```diff
@@ -298,6 +312,209 @@ private void DoStatusUpdate(object sender, StatusUpdateArgs args)
 	    }
 
 
+        private void CmdrJournalParser_CommitCrimeEvent(object sender, CommitCrimeLog eventData) {
+            var msg = new TPAMessage()
+            {
+                action = "CommitCrime",
+                data = new JObject
+                (
+                    new JProperty("CrimeType", eventData.CrimeType),
+                    new JProperty("RatID", _myplayer.RatId.FirstOrDefault()),
+                    new JProperty("CurrentSystem", MyPlayer.CurrentSystem),
+                    new JProperty("RescueID", MyClient.Rescue.id)
+                )
+            };
+
+            if (eventData.Victim != null)
+                msg.data.Add("Victim", eventData.Victim);
+
+            _apworker.SendTpaMessage(msg);
+        }
+
+        private void CmdrJournalParser_DiedEvent(object sender, DiedLog eventData)
+        {
+            if (MyClient?.Rescue != null)
+            {
+                _apworker.SendTpaMessage(new TPAMessage()
+                {
+                    action = "RatDeath",
+                    data = new JObject
+                    (
+                        new JProperty("Killers", eventData.KillersList?.Select(k => k.Name).ToArray()),
+                        new JProperty("RatID", _myplayer.RatId.FirstOrDefault()),
+                        new JProperty("CurrentSystem", MyPlayer.CurrentSystem),
+                        new JProperty("RescueID", MyClient.Rescue.id)
+                    )
+                });
+                AppendStatus("Sending death notice.");
+            }
+        }
+
+	    private void _cmdrJournalParser_EscapeInterdictionEvent(object sender, EscapeInterdictionLog eventData)
+	    {
+	        if (MyClient?.Rescue != null)
+	        {
+	            _apworker.SendTpaMessage(new TPAMessage()
+	            {
+	                action = "Interdiction:Update",
+	                data = new JObject
+	                (
+	                    new JProperty("Interdiction", "Escaped"),
+	                    new JProperty("Interdictor", eventData.Interdictor),
+	                    new JProperty("RatID", _myplayer.RatId.FirstOrDefault()),
+	                    new JProperty("CurrentSystem", MyPlayer.CurrentSystem),
+	                    new JProperty("RescueID", MyClient.Rescue.id)
+	                )
+	            });
+                AppendStatus("Sending Escape Interdiction Noticiation.");
+            }
+	    }
+
+	    private void CmdrJournalParser_FsdJumpEvent(object sender, FsdJumpLog eventData)
+        {
+            TriggerSystemChange(eventData.StarSystem);
+        }
+
+        //TODO Move this... somewhere
+        private DateTime lastHullDamageEvent;
+	    private void CmdrJournalParser_HullDamageEvent(object sender, HullDamageLog eventData)
+	    {
+            if(MyClient?.Rescue != null) {
+                if ((eventData.Timestamp - lastHullDamageEvent).TotalMinutes < 1)
```

Comment:
Right now the "UnderAttack" is limited to a 1 minute cool-down from the first message. should be be adjusted?